handlers/create.go

```diff
@@ -26,6 +27,7 @@ const (
 )
 
 var RegExDockerMsg = regexp.MustCompile("msg=.*")
+var RegExHyphen = regexp.MustCompile(`([a-z])([A-Z])`)
```

Comment:
These two vars (RegExDockerMsg and RegExHyphen) should probably be lowercase so they are not exported.